src/includes/getting-started-verify/ruby.mdx

```diff
@@ -1,9 +1,5 @@
-The simplest way to verify your setup is to create an exception:
+The simplest way to verify your setup is to send a message event:
```

Comment:
Note that is line is redundant to what's in the common template. I'd suggest removing it.